modules/about-etcd-encryption.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * authentication/encrypting-etcd.adoc
+
+[id="about-etcd_{context}"]
+= About etcd encryption
+
+By default, etcd data is not encrypted in {product-title}. You can enable etcd encryption for your cluster to provide an additional layer of protection against data leakage. For example, it protects the loss of sensitive data if an etcd backup is exposed to the incorrect parties.
```

Comment:
"data leakage" is a bit jargony and will probably not translate well. I might just say that it provide an additional later of data security.